quipucords/api/messages.py

```diff
@@ -140,7 +140,7 @@
 # Scan Manager/Signal
 SIGNAL_STATE_CHANGE = 'SIGNAL %s received for scan job.'
 SIGNAL_SCAN_MANAGER_CRASH = 'Process unexpectedly crashed.  See logs.'
-SIGNAL_SCAN_MANAGER_RESTART = 'Recovering manager..'
+SIGNAL_SCAN_MANAGER_RESTART = 'Recovering manager.'
```

Comment:
Copyright should be updated to have 2017-2018 date.

Reply:
fixed 👍